### ISSUE TITLE
Possible python scripst improvements?

### ISSUE THREAD

**Issue by blattms** [OPM/opm-reference-manual#209]
- Would be cool to have a working default for `--maindir` if the scripts are called from the root directory of the git repo checkout
- `--filename` option is misleading as it actually needs a path. Maybe it is better to skip name and just use `--file`?

**Comment by blattms** [comment]
Why a user cd into a subirectory to  call  a script?
So the filename is a path relative to maindir?

**Comment by hakonhagland** [comment]
Good point. The idea was that these scripts might not be something that you want to install globally on your system since they are only useful inside the opm-reference-manual repository. So then the user should rather activate the poetry virtual environment before running the scripts.

**Comment by hakonhagland** [comment]
I agree that the scripts behave strangely/unexpectedly if installed globally. I will try to fix it.
